sdk/src/main/java/com/mapbox/maps/plugin/MapPluginRegistry.kt

```diff
@@ -157,7 +157,7 @@ internal class MapPluginRegistry(
         cameraState.pitch,
         cameraState.bearing,
         cameraState.padding.let { insets ->
-          arrayOf(insets.left, insets.top, insets.right, insets.bottom)
+          arrayOf(insets.top, insets.left, insets.bottom, insets.right)
```

Comment:
We should update the documentation to be specific as well, this might affect other codebase as well, e.g weather cc @paulrenn67 
consider the potential impact to users who implement their own MapCameraPlugin, I'd vote for keeping the original order.

Reply:
Maybe we should just keep the original order and add specific documentation on the meaning of each element in the list?

Reply:
Yes, this is indeed the behavioural change. Agree it's better to keep the original order, will add the docs and revert the change.